### PR TITLE
Cleanup: Don't use get_val<> in stream line

### PR DESCRIPTION
**Pull request by shinobu-x**
Signed-off-by: Shinobu Kinjo <shinobu@redhat.com>

### CODE REVIEW

File: src/mon/MgrMonitor.cc [hunk 1]
```diff
@@ -580,8 +580,8 @@ void MgrMonitor::tick()
   if (!pending_map.available &&
       !ever_had_active_mgr &&
       should_warn_about_mgr_down() != HEALTH_OK) {
-    dout(10) << " exceeded mon_mgr_mkfs_grace "
-             << g_conf->get_val<int64_t>("mon_mgr_mkfs_grace")
+    auto mon_mgr_mkfs_grace = g_conf->get_val<int64_t>("mon_mgr_mkfs_grace");
+    dout(10) << " exceeded mon_mgr_mkfs_grace " << mon_mgr_mkfs_grace
```
Comment (tchaikov) [inline src/mon/MgrMonitor.cc]: does this make any difference?

Comment (shinobu-x) [reply src/mon/MgrMonitor.cc]: Using get_val<> wouldn't cheap, isn't it?

Comment (tchaikov) [reply src/mon/MgrMonitor.cc]: and

- in the worst case, if debug-mon>=10, your change does not improve the performance. as `get_val<>` always gets called
- in the best case, if debug-mon<10, your change hurts the performance, the return value of `get_val<>()` is dropped on the floor, and is not used.

Comment (shinobu-x) [reply src/mon/MgrMonitor.cc]: i got your point now. i didn't properly understand the context of your comment. sorry for that...

File: src/os/bluestore/BlueStore.cc [hunk 2]
```diff
@@ -4719,8 +4719,10 @@ int BlueStore::_open_db(bool create, bool to_repair_db)
                << fn + ".slow" << ","
                << (uint64_t)(slow_size * 95 / 100);
       cct->_conf->set_val("rocksdb_db_paths", db_paths.str(), false);
-      dout(10) << __func__ << " set rocksdb_db_paths to "
-	       << cct->_conf->get_val<std::string>("rocksdb_db_paths") << dendl;
+      auto rocksdb_db_paths = cct->_conf->get_val<std::string>(
+        "rocksdb_db_paths");
+      dout(10) << __func__ << " set rocksdb_db_paths to " << rocksdb_db_paths
```
Comment (tchaikov) [inline src/os/bluestore/BlueStore.cc]: what's the merit of this change?

**Pull request closed without merging**